tutor/templates/config/defaults.yml

```diff
@@ -2,6 +2,9 @@
 # This file includes all Tutor setting defaults. Settings that do not have a
 # default value, such as passwords, should be stored in base.yml.
 # This must be defined early
+ATLAS_REVISION: "{% if OPENEDX_COMMON_VERSION == 'master' %}main{% else %}{{ OPENEDX_COMMON_VERSION }}{% endif %}"
```

Comment:
Do you expect to maintain "open-release/xxx.master" branches and "open-release/xxx.y" tags? I see that the openedx-translations repo does not have an [openedx.yaml](https://github.com/openedx/edx-platform/blob/master/openedx.yaml) file.

Reply:
> Do you expect to maintain "open-release/xxx.master" branches and "open-release/xxx.y" tags? I see that the openedx-translations repo does not have an [openedx.yaml](https://github.com/openedx/edx-platform/blob/master/openedx.yaml) file.

Not really.

How do you recommend we proceed with this variable?